disnake/invite.py

```diff
@@ -275,32 +275,34 @@ class Invite(Hashable):
-    |                                    | or valid event ID in URL or invite object                         |
-    +------------------------------------+-------------------------------------------------------------------+
-
-    If it's not in the table above then it is available by all methods.
+    +------------------------------------+---------------------------------------------------------------------+
+    |             Attribute              |                          Method                                     |
+    +====================================+=====================================================================+
+    | :attr:`max_age`                    | :meth:`abc.GuildChannel.invites`\\, :meth:`Guild.invites`            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`max_uses`                   | :meth:`abc.GuildChannel.invites`\\, :meth:`Guild.invites`            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`created_at`                 | :meth:`abc.GuildChannel.invites`\\, :meth:`Guild.invites`            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`temporary`                  | :meth:`abc.GuildChannel.invites`\\, :meth:`Guild.invites`            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`uses`                       | :meth:`abc.GuildChannel.invites`\\, :meth:`Guild.invites`            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`approximate_member_count`   | :meth:`Client.fetch_invite` with ``with_counts`` enabled            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`approximate_presence_count` | :meth:`Client.fetch_invite` with ``with_counts`` enabled            |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`expires_at`                 | :meth:`Client.fetch_invite` with ``with_expiration`` enabled        |
+    +------------------------------------+---------------------------------------------------------------------+
+    | :attr:`guild_scheduled_event`      | :meth:`Client.fetch_invite` with valid ``guild_scheduled_event_id`` |
+    |                                    | or valid event ID in URL or invite object                           |
+    +------------------------------------+---------------------------------------------------------------------+
+
+    If something is not in the table above, then it's available by all methods.
 
     Attributes
```

Comment:
`max_age` is missing `Optional[]` type below

Reply:
good catch, ty! https://github.com/DisnakeDev/disnake/pull/864/commits/9607d8af10a70b636e8f39344507a102570e832d